content/nim/deploy/kubernetes/deploy-using-helm.md

```diff
@@ -149,34 +154,27 @@ imagePullSecrets:
 apigw:
   image:
     repository: private-registry.nginx.com/nms/apigw
-    tag: <version>
 core:
   image:
     repository: private-registry.nginx.com/nms/core
-    tag: <version>
 dpm:
   image:
     repository: private-registry.nginx.com/nms/dpm
-    tag: <version>
 ingestion:
   image:
     repository: private-registry.nginx.com/nms/ingestion
-    tag: <version>
 integrations:
   image:
     repository: private-registry.nginx.com/nms/integrations
-    tag: <version>
 secmon:
   image:
     repository: private-registry.nginx.com/nms/secmon
-    tag: <version>
 utility:
   image:
     repository: private-registry.nginx.com/nms/utility
-    tag: <version>
 ```
 
-These values are required when pulling images from the NGINX private registry. The chart doesn't auto-resolve image tags. Set each `tag:` value to match the NGINX Instance Manager version you want to install. Refer to the Helm chart table for version details.
+The values required to pull images from the NGINX private registry are now automatically resolved, including image tags. Each image’s `tag:` is set by default to the latest NGINX Instance Manager version. See the Helm chart table for version details.
```

Comment:
`See the Helm chart table for version details`

Can this be linked to the table?

Reply:
```suggestion
The values required to pull images from the NGINX private registry are now automatically resolved, including image tags. Each image’s `tag:` is set by default to the latest NGINX Instance Manager version. See the [Helm chart table in the chart installation section]({{< ref "/nim/deploy/kubernetes/deploy-using-helm.md#install-the-chart" >}}) for version details.
```